set.mm

```diff
@@ -17556,6 +17556,15 @@ variables and no wff metavariables (see ~ ax12wdemo for an example of
       ( hbaevg ) ABCBAD $.
   $}
 
+  ${
+    $d x y $.  $d u t z $.
+    $( Any variable is free in ` -. A. x x = y ` , if ` x ` and ` y ` are
+       distinct.  (Contributed by Wolf Lammen, 9-Apr-2021.) $)
```

Comment:
Is the version without DV(x,y) in set.mm ?  It would be worth adding it, with cross-links between the two.

Reply:
it is (hbnae).  links added